### ISSUE TITLE
homebridge don´t start with node 12.3.1 

### ISSUE THREAD

**Issue by Krocko** [chrisjshull/homebridge-nest#95]
After updating node to v.12.3.1 this plugin don't work anymore.
Homebridge won´t start.
I got the following error:

```
Jun 02 09:30:52 raspberrypi systemd[1]: homebridge.service: Main process exited, code=killed, status=11/SEGV
Jun 02 09:30:52 raspberrypi systemd[1]: homebridge.service: Unit entered failed state.
Jun 02 09:30:52 raspberrypi systemd[1]: homebridge.service: Failed with result 'signal'.
```

If i go back to node v.12.1.0 this plugin is working again.

Can you fix this @chrisjshull?

**Comment by chrisjshull** [comment]
https://github.com/chrisjshull/homebridge-nest/blob/master/README.md#need-new-owner

**Comment by jankais3r** [comment]
Would be cool to have this plugin working again. I just reinstalled homebridge on new raspberry with up-to-date Node and suddenly it segfaults...

**Comment by adriancable** [comment]
jankais3r/Krocko - I believe homebridge-nest-native works on Node 12.6.0. Please confirm then we can close this. Chris - I am happy to take ownership of this repo - please transfer on github and npmjs to me at your convenience.

**Comment by Krocko** [comment]
Hi Adrian. Your Plugin working perfect with Node 12.6.0.

**Comment by adriancable** [comment]
Krocko - great. I think the issue was incompatibility between Firebase and the latest node. But, I do not use Firebase in my version of the plug-in.

I've let Chris Shull know I'm happy to take over the repo on github/npm so as soon as he has the chance to do that, we all should be good to go. I'll publish as homebridge-nest@3.0.0.

**Comment by Krocko** [comment]
I think, the Signal package was the problem.

**Comment by adriancable** [comment]
Fixed in 3.0.0.
